src/mon/MonCap.h

```diff
@@ -72,11 +73,16 @@ struct MonCapGrant {
    *      of key/value pairs that constrain use of that command.  if no pairs
    *      are specified, any arguments are allowed; if a pair is specified, that
    *      argument must be present and equal or match a prefix.
+   *
+   *  - an fs name ('allow fsname foo')
```

Comment:
s/an/a/

Reply:
Missed out replying to this, "an"  is correct when `F` is first letter of an acronym (eg, "an FBI agent").